### ISSUE TITLE
SQLite FTS support

### ISSUE THREAD

**Issue by preetb123** [yahoo/squidb#204]
is there any way i can create FTS tables using squidb. If not then how can i achieve full text searching?

**Comment by jdkoren** [comment]
Absolutely! In the `@TableModelSpec` annotation, you can specify the module name for virtual tables (which include FTS):

```java
@TableModelSpec(className="FtsModel", tableName="fts_models", moduleName="FTS4")
public class FtsModelSpec {
    // ...
}
```

When the annotation processor generates the model class, the static final `TABLE` field will be of type [`VirtualTable`](https://github.com/yahoo/squidb/blob/master/squidb/src/com/yahoo/squidb/sql/VirtualTable.java) instead of `Table`.

**Comment by preetb123** [comment]
thanks @jdkoren.
